lib/src/edhoc.rs

```diff
@@ -45,7 +45,7 @@ pub fn edhoc_key_update(
         _th_3,
     ) = state;
 
-    let mut prk_new_buf: BytesMaxBuffer = [0x00; MAX_BUFFER_LEN];
+    let mut prk_new_buf: BytesMaxBuffer;
```

Comment:
This one we may have to revert, as `hax` does not like non-initialized variables.

Reply:
I think the preferred fix here is to not have a single buf but two bufs, not mutate them, and rely on the compiler to see that those two are not used at the same time and thus don't have to sit next to each other on the stack. Proposing the new fix (forcpushibly replacing the merge, which did both a merge and a fix at the same time)